app/lib/actions/pulp3/repository/update_cv_repository_cert_guard.rb

```diff
@@ -0,0 +1,19 @@
+module Actions
+  module Pulp3
+    module Repository
+      class UpdateCVRepositoryCertGuard < Pulp3::Abstract
+        def plan(repository, _smart_proxy)
+          plan_self(:root_id => repository.root.id)
+        end
+
+        def run
+          root = ::Katello::RootRepository.find(input[:root_id])
+          cv_repositories = root.repositories - [root.library_instance]
+          cv_repositories.each do |repo|
+            ForemanTasks.async_task(::Actions::Pulp3::Repository::RefreshDistribution, repo, SmartProxy.pulp_master)
```

Comment:
since '::Actions::Katello::Repository::UpdateCVRepoCertGuard' is being run async anyways, we can just call refresh distribution as part of a 'plan_task' within this action's plan phase.  No need to spin them all off as async_tasks